app/questionnaire/placeholder_transforms.py

```diff
@@ -287,6 +287,7 @@ def get_ordinal_indicator(self, number_to_format: int) -> str:
                 17: "eg",
                 19: "eg",
             }.get(number_to_format, "fed")
+        raise NotImplementedError
```

Comment:
New line for readibility and text for context if the error were to happen.

```suggestion
        
        raise NotImplementedError(f"Language code '{self.language}' not implemented.")
```